azure-pipelines.yml

```diff
@@ -0,0 +1,85 @@
+# Python package
+# Create and test a Python package on multiple Python versions.
+# Add steps that analyze code, save the dist with the build record, publish to a PyPI-compatible index, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/python
+
+trigger:
+  tags:
+    include:
+      - '*'
+  branches:
+    include:
+      - master
+      - release/*
+      - refs/tags/*
+pr:
+  - master
+  - release/*
+
+jobs:
+  - job: pytest
+    # how long to run the job before automatically cancelling
+    timeoutInMinutes: 45
+    # how much time to give 'run always even if cancelled tasks' before stopping them
+    cancelTimeoutInMinutes: 2
+
+    pool: gridai-spot-pool
+
+    # ToDo: this need to have installed docker in the base image...
+    #container: "pytorchlightning/pytorch_lightning:base-cuda-py$[ variables['python.version'] ]-torch1.6"
+    container:
+      image: "pytorchlightning/pytorch_lightning:base-cuda-py3.7-torch1.6"
+      #endpoint: azureContainerRegistryConnection
+      options: "--runtime=nvidia -e NVIDIA_VISIBLE_DEVICES=all"
+
+    workspace:
+      clean: all
+
+    steps:
+
+    - bash: |
+        lspci | egrep 'VGA|3D'
+        whereis nvidia
```

Comment:
it seems like `whereis nvidia` always echos:
```
nvidia:
```
https://dev.azure.com/PytorchLightning/pytorch-lightning/_build/results?buildId=8250&view=logs&j=3afc50db-e620-5b81-6016-870a6976ad29&t=b8e11f49-5cfc-56d8-ba3e-ec2e1e8ddf1d&l=13